beta/docs/recipes/next/integrate.md

```diff
@@ -0,0 +1,93 @@
+---
+id: nextjs
```

Comment:
please, do not "just copy" original files. Beta version has differences of the structure of a document.

For example:

`id` is not required anymore. But Title must also be in place as `h1` header.
https://github.com/effector/effector/blob/master/beta/docs/recipes/react/effects.md?plain=1#L1-L6

https://github.com/effector/effector/blob/master/docs/recipes/react/example-effects.md?plain=1#L1-L5

Also the structure of some documentation links was change. Please, run the beta documentation and check each article for corectness

Reply:
done